x/txfees/keeper/mempool-1559/code.go

```diff
@@ -65,22 +78,27 @@ func (e *EipState) startBlock(height int64) {
 		e.CurBaseFee = e.tryLoad()
 	}
 
+	// we reset the CurBaseFee every ResetInterval
 	if height%ResetInterval == 0 {
 		e.CurBaseFee = DefaultBaseFee.Clone()
 	}
 }
 
+// deliverTxCode runs on every transaction in the feedecorator ante handler and sums the gas of each transaction
 func (e *EipState) deliverTxCode(ctx sdk.Context, tx sdk.FeeTx) {
 	if ctx.BlockHeight() != e.lastBlockHeight {
 		fmt.Println("Something is off here? ctx.BlockHeight() != e.lastBlockHeight", ctx.BlockHeight(), e.lastBlockHeight)
 	}
 	e.totalGasWantedThisBlock += int64(tx.GetGas())
-	// fmt.Println("height, tx gas, blockGas", ctx.BlockHeight(), tx.GetGas(), e.totalGasWantedThisBlock)
```

Comment:
Another print above could be replaced with `ctx.Logger().Error()`

Reply:
Good idea updated